pygitguardian/sca_models.py

```diff
@@ -33,6 +33,8 @@ class SCAIgnoredVulnerability(Base, FromDictMixin):
 class SCAScanParameters(Base, FromDictMixin):
     minimum_severity: Optional[str] = None
     ignored_vulnerabilities: List[SCAIgnoredVulnerability] = field(default_factory=list)
+    ignore_no_fix: bool = False
+    ignore_fix_available: bool = False
```

Comment:
couldn't we set it as Optional[bool] = None ?

Reply:
As if we do not specify it, we want to include the vulnerability, it's like having `False` as a default value. I don't think it brings anything to allow `None` while it will have the same behavior as `False`.

Reply:
Good catch, that's an old typo !

Reply:
ok, can you add the typing for these fields please?